packages/flutter_tools/lib/src/commands/build_aot.dart

```diff
@@ -335,6 +335,7 @@ Future<String> _buildAotSnapshot(
       extraFrontEndOptions: extraFrontEndOptions,
       linkPlatformKernelIn : true,
```

Comment:
[nit] Remove spaces before :
This line and the next.

Reply:
done